addons/test_discuss_full/tests/test_performance.py

```diff
@@ -107,6 +107,7 @@ def test_init_messaging(self):
             'starred_counter': 1,
             'channels': [
                 {
+                    'authorized_group': 'User types / Internal User',
```

Comment:
Prefer reading the value from the group in code rather than hardcoding it, the goal of the test is not to ensure the group has this name exactly.